server/models/searches.js

```diff
@@ -2,86 +2,36 @@ var mongoose = require('mongoose');
 	
 	
 var searchSchema = new mongoose.Schema({	
+    ids: {
+		type: [String]
+	},
     names: {
-		type: [[name]]
+		type: {
+			value: String,
+    		label: String,
+    		First: String,
+    		Last: String,
+		},
 	},
 	services: { 
-		type: [[service]]
+		type: [String]
 	},
 	location: {
-		type: {locationSchema}
-	},
-	time: {
-		type: {time}
-	},
-	timeOfDay: {
-		type: {timeOfDay}
-	},
-	numSessions: {
-		type: {numSession}
-	}
-}, { collection: 'searches' });
-
-var name = new mongoose.Schema({
-	first: {
-		type: String
-	},
-	last: {
 		type: String
 	},
-	label: {
-		type: String
-	},
-	value: {
-		type: String
-	}
-});
-
```

Comment:
```
var name = new mongoose.Schema({	
	first: {	
		type: String	
	},	
	last: {	
		type: String			
	},		},
	label: {		
		type: String	
	},	
	value: {	
		type: String	
	}	
}); 
```
can be changed to... (like you had for the ClinicianSchema)
```
var name = new mongoose.Schema({	
  LastName: {
    type: String,
    required: true,
  },
  FirstName: {
    type: String,
    required: true,
  },
}); 
```
This can be used on line 8 of searches.js and line 4-10 of clinician schema.

Reply:
The `names` property (with `label` and `value` as well) on the search schema is currently used by the View page to request the clinicians' schedules and calculate their available times. We can identify the clinicians using their IDs so once the calculation is moved to the backend we won't need the names property on the search anymore.

I've instead just set the type of the names property to be an array of the previous type.

Reply:
Works for me!